multigeiger/multigeiger.ino

```diff
@@ -274,16 +277,16 @@ void loop() {
   if (Serial_Print_Mode == Serial_Statistics_Log)
     statistics_log(gm_counts, gm_count_time_between);
 
-  transmit(current_ms, gm_counts, gm_count_timestamp, hv_pulses, have_thp, temperature, humidity, pressure);
+  int wifi_status = update_wifi_status();
+
+  transmit(current_ms, gm_counts, gm_count_timestamp, hv_pulses, have_thp, temperature, humidity, pressure, (wifi_status == ST_WIFI_CONNECTED));
```

Comment:
I'ld rather put the `if` here. See also my comments in the ticket.

Reply:
hmm, no, it's more difficult.

transmit needs (usually) either lora or wifi, in theory it could even use wifi and lora.